### DISCUSSION TITLE
[Help] `{{tags}}` separator in `--format`

### DISCUSSION THREAD

**Discussion by mawkler** [zk-org/zk#343] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

When list a note with multiple tags like so:

```sh
zk list --format "{{title}}, tags: [{{tags}}]"
```

notes with multiple tags have no separators between the tags.

For instance, the following note:

```markdown
---
title: Test note
tags: [foo, bar, baz]
---
```

gets formatted like this:

```
Test note, tags: [foobarbaz]
```

I would like it to be formatted like this:

```
Test note, tags: [foo, bar, baz]
```

How do I input a separator like `, ` between the tags?

**Comment by mickael-menu** [comment, marked as answer]
`tags` is a list of strings, you can iterate over it:

```handlebars
{{#each tags}}{{this}}{{#if @last}}{{else}}, {{/if}}{{/each}}
```

[You can learn more about the templating system here](https://handlebarsjs.com/guide/).

**Reply by mawkler** [reply]
Thank you! I missed the whole templating syntax when reading the docs. My mistake!
